src/app/auth/auth.component.ts

```diff
@@ -224,13 +222,19 @@ export class AuthComponent implements OnInit, OnDestroy {
    * Handles the clean up required after authentication and setting up of user data is completed.
    */
   handleAuthSuccess() {
-    this.titleService.setTitle(appSetting.name
+    this.setTitle();
+    this.router.navigateByUrl(this.phaseService.currentPhase);
+    this.authService.changeAuthState(AuthState.Authenticated);
+  }
+
+  setTitle(): void {
+    const appSetting = require('../../../package.json');
+    const title = appSetting.name
       .concat(' ')
       .concat(appSetting.version)
       .concat(' - ')
-      .concat(this.phaseService.getPhaseDetail()));
-    this.router.navigateByUrl(this.phaseService.currentPhase);
-    this.authService.changeAuthState(AuthState.Authenticated);
+      .concat(this.phaseService.getPhaseDetail());
```

Comment:
Perhaps we could use string interpolation to more clearly define the title string instead of using `.concat()`. So something along the lines of

```typescript
const title = `${appSetting.name} ${appSetting.version} - ${this.phaseService.getPhaseDetail()}`
```

It might be a little cleaner and also no need to add additional `.concat()`s for the spaces and dashes. What do you think?

Reply:
Yup, that sounds good!